wagtail/wagtailsearch/index.py

```diff
@@ -168,6 +168,9 @@ def get_value(self, obj):
         if isinstance(field, RelatedField):
             return getattr(obj, self.field_name)
 
+        if isinstance(field, ForeignObjectRel):
+            return getattr(obj, self.field_name)
```

Comment:
This can be merged with the previous condition:
```python
if isinstance(field, (RelatedField, ForeignObjectRel)):
    return getattr(obj, self.field_name)
```